### ISSUE TITLE
Internal list that contains all devices that are emitting a bluetooth signal

### ISSUE THREAD

**Issue by Meisterlu** [amosproj/amos2022ss05-find-my-hearing-aid#72]
## User story
1. As a user 
2. I want / need to have a list of surrounding BLE devices 
3. So that So that I can see all bluetooth devices that are emitting a bluetooth signal and are surrounding me 

## Acceptance criteria
- [ ] extracted list of devices in surrounding that emit a bluetooth signal
- [ ] Show list of devices in code or terminal

**Comment by NicoStellwag** [comment]
We kind of already had that.
What we changed is that devices now show up in the gui right when they are detected (in contrast to only after the scanning period is over).  This should make the ux a bit better and it allows us to stop scanning when a user already selected a device.

**Comment by Meisterlu** [comment]
@NicoStellwag Alright. I agree definitely improves the UX! Next step would have been to show human readable names, but that's already finished.

**Comment by Chippelius** [comment]
@domi1504 @NicoStellwag I need the results of this issue in order to work on issue #80. Now there are two options: 

1. I can pull the changes directly from the branch of this issue. When I'm done, I will create a pull request into dev containing both, my changes and yours. In order to avoid merge conflicts it is important that you do not create a pull-request of your own in this case! (Pull-requests are what is listed in the release notes of the sprint release. I don't know how much you care about reading your name there, which is why I'm asking beforehand.) 
2. I can pull the changes from the dev branch after you created a pull-request that has been merged successfully. This would require you to ask the release manager (which is Marib for this sprint) to review and merge your pull-request before tomorrow afternoon, so I still have time to work on my own issue before the pull-request deadline. 

Which one would you prefer? (If your changes are not merged into dev by tomorrow, 4 pm, and you did not answer here, I'm assuming you want to go with option 1.)
